src/modules/Builder/components/Configuration/components/Wireless/Wireless.jsx

```diff
@@ -121,66 +48,91 @@ export default class Wireless extends Component {
-                        <Form.Group>
-                            <Form.Label>Network name</Form.Label>
-                            <Input
-                                id="SSIDID"
-                                name="SSID"
-                                type="text"
-                                value={SSID}
-                                onChange={this.onChangeData}
-                                disabled={!enableWireless}
-                                validationFunc={this.SSIDValidor}
-                            />
-                            <div className="error invalid-feedback">
-                                Network name must be between 1 and 31 characters of Latin letters,
-                                numbers or a dot character
-                            </div>
-                        </Form.Group>
-                        <Form.Group>
-                            <Form.Label>Password</Form.Label>
-                            <Input
-                                id="PSKID"
-                                name="PSK"
-                                type="password"
-                                value={PSK}
-                                onChange={this.onChangeData}
-                                disabled={!enableWireless}
-                                validationFunc={this.PSKValidor}
-                            />
-                            <div className="error invalid-feedback">
-                                Password must be either empty or between 8 and 63 characters
-                                of Latin letters, numbers or a dot character
-                            </div>
-                        </Form.Group>
-                    </div>
-                </Card.Body>
-            </Card>
-        );
-    }
-}
+    };
+
+    return (
+        <Card>
+            <Card.Body>
+                <Form>
+                    <Form.Group>
+                        <Form.Check
+                            type="checkbox"
+                            id="show-ordinary-password"
+                            label="Support Wi-Fi"
+                            onChange={() => updateConfigurationParamsAction(
+                                "enable_wireless", !configurationParams.enable_wireless,
+                            )}
+                            checked={configurationParams.enable_wireless || false}
+                        />
+                    </Form.Group>
+                    <Form.Group className="position-relative">
+                        <Form.Label>Network name</Form.Label>
+                        <Form.Control
+                            as={Input}
+                            id="WPA_SSIDID"
+                            name="WPA_SSID"
+                            type="text"
+                            value={configurationParams.WPA_SSID}
+                            onChange={(name, value) => updateConfigurationParamsAction(name, value)}
```

Comment:
Don't wrap `updateConfigurationParamsAction` in arrow function. Pass `updateConfigurationParamsAction` as `onChange` value.